### PR TITLE
MOSIP-10568, MOSIP-10271: Template bug fix and RefNumber added for Documents

### PR DESCRIPTION
**Pull request by HimajaDhanyamraju2**
Documents

### CODE REVIEW

File: registration/registration-client/src/main/java/io/mosip/registration/controller/reg/DocumentScanController.java [hunk 1]
```diff
@@ -407,7 +419,42 @@ private void initialize() {
...
+				documentVBoxes.put(documentCategory.getId(), documentVBox);				
+				
+				VBox refNumVBox = new VBox();
+				refNumVBox.setId(docCategoryCode + "RefNumVBox");
+				refNumVBox.getStyleClass().add(RegistrationConstants.SCAN_VBOX);
+				TextField numberOfDocs = new TextField();
+				numberOfDocs.setId(docCategoryCode + "RefNum");
+				numberOfDocs.setPromptText("Ref Number");
+				numberOfDocs.getStyleClass().add(RegistrationConstants.DEMOGRAPHIC_TEXTFIELD);
+				numberOfDocs.setStyle("-fx-font-size:13");
```
Comment (ghost) [inline registration/registration-client/src/main/java/io/mosip/registration/controller/reg/DocumentScanController.java]: @HimajaDhanyamraju2  Can you pls add a separate styleclass and move the fontsize styling under the style class

File: registration/registration-services/src/main/java/io/mosip/registration/util/acktemplate/TemplateGenerator.java [hunk 2]
```diff
@@ -805,20 +810,29 @@ private void setUpDemographicInfo(RegistrationDTO registration, Map<String, Obje
 		Map<String, Object> defaultDemographics = registration.getDefaultDemographics();
 		if (defaultDemographics != null && !defaultDemographics.isEmpty()) {
 			templateValues.put(RegistrationConstants.TEMPLATE_APPLICANT_NAME_PRIMARY_LABEL,
-					applicationLanguageProperties.getString("fullName"));
+					applicationLanguageProperties.getString("applicantname"));
```
Comment (ghost) [inline registration/registration-services/src/main/java/io/mosip/registration/util/acktemplate/TemplateGenerator.java:813]: How are we taking applicant name here @HimajaDhanyamraju2  ?

File: registration/registration-client/src/main/java/io/mosip/registration/controller/reg/DocumentScanController.java (hunk 1 above)
Comment (HimajaDhanyamraju2) [reply registration/registration-client/src/main/java/io/mosip/registration/controller/reg/DocumentScanController.java]: Modified as suggested

File: registration/registration-services/src/main/java/io/mosip/registration/util/acktemplate/TemplateGenerator.java (hunk 2 above)
Comment (HimajaDhanyamraju2) [reply registration/registration-services/src/main/java/io/mosip/registration/util/acktemplate/TemplateGenerator.java:813]: Applicant name is stored in the session based on the group "FullName" in schema.

**Pull request merged**